CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Fixes
 
 - Faucet webpage is missing `background.png` and `favicon.ico` (#672).
+- Add wrapper for `Mmr` (#668).
```

Comment:
We can leave this out of the changelog; this is essentially an internal change only.